api_gateway.tf

```
@@ -0,0 +1,60 @@
+resource "aws_api_gateway_rest_api" "dns-manager" {
+  name                     = "dns-manager"
+  description = "API for dns-manager lambda function"
+  api_key_source           = "HEADER"
+  binary_media_types       = []
+  minimum_compression_size = -1
+  tags                     = {}
+
+  endpoint_configuration {
+    types = ["REGIONAL"]
+  }
+}
+
+resource "aws_api_gateway_method" "post" { 
+  resource_id          = "${aws_api_gateway_rest_api.dns-manager.root_resource_id}"
```

Please, use tf 0.12 notation in all code.
```suggestion
  resource_id          = aws_api_gateway_rest_api.dns-manager.root_resource_id
```

---

> Please, use tf 0.12 notation in all code.

Will not work, see https://shalb.slack.com/archives/CRFSNTDGX/p1587120390009000?thread_ts=1587117760.001000&cid=CRFSNTDGX

---

As you test, it still works :)